src/open_inwoner/templates/pages/cases/document_form.html

```diff
@@ -5,5 +5,6 @@
     {% csrf_token %}
     {% input form.type no_label=True no_help=True class="label input" id="id_type" extra_classes="file-type__select" %}
     {% file_input form.files %}
-    {% form_actions primary_text=_("Upload documenten") primary_icon="arrow_forward" enctype="multipart/form-data" %}
+    {% form_actions primary_text=_("Upload documenten") enctype="multipart/form-data" %}
+    <p class="p--upload-error p--small p--upload-info p--upload-help">{% trans "Verwijder eerst bestanden die niet voldoen aan de voorwaarden" %}</p>
```

Comment:
A class `p--upload-error` is being referenced here which should be (see previous comment):

- An actual modifier of "P", which could be correct but in which case it's specified in the wrong file and with an to broad modifier name.
- An element of file-input in which case it can/should not be used outside the template for the file input.
- A component (block) with it's own scss file (and possbly template) in which case it's interface and meaning should be properly defined.